README.md

```diff
@@ -15,8 +15,15 @@ L'installation se fait avec le package `devtools`:
 ```r
 devtools::install_github("signaux-faibles/rsignauxfaibles")
 ```
```

Comment:
J'ai tenté d'importer le package comme indiqué mais j'ai l'erreur suivante : 
devtools::install_github("signaux-faibles/rsignauxfaibles")
Error: Failed to install 'rsignauxfaibles' from GitHub:
  Could not resolve host: api.github.com; Unknown error
Je suppose que c'est parce que je n'ai pas les accès d'ouvert sur mon poste Banque de France ....

Reply:
Oui c'est l'accès internet depuis la plateforme !